encryption.rst

```diff
@@ -223,3 +360,24 @@ Running using an environment variable
 .. code::
 
    $ {ENVPREFIX}_ENCRYPTION_PEM_PATH=rsa_pri.pem {command} run encrypted.sif
+
+.. note:: 
+
+   When ``run``, ``shell``, or ``exec`` an ``Gocryptfs squashfs`` encrypted image with {Project}
```

Comment:
This also applies to ``instance start``, right?  So maybe it would be better to just use a generic term like "executing".   Use {aProject} here instead of {Project}.

Reply:
thanks for the comments. reflected in this commit https://github.com/apptainer/apptainer-userdocs/pull/179/commits/c7f08f6f219f7edc65f6a48fdae506166718c9e3